src/index.ts

```diff
@@ -27,6 +28,10 @@ export class WaxJS {
   private readonly feeFallback: boolean;
   private readonly metricURL: string;
   private readonly returnTempAccounts: boolean;
+  private chainName: string | null;
+  private chainId: string | null;
+  private lightApiEndpoint: string | null;
```

Comment:
why are we adding the lightApiEndpoint in here? I don't see it used anywhere else in the library

Reply:
@porkchop we have fetching data on UI side which need to specify lightAPI (this changed base on testnet or mainnet)

Reply:
ok thanks @sonnguyenwax